### PR TITLE
add extension authorization doc

### PR DESCRIPTION
**Pull request by zhou1203**

### CODE REVIEW

File: sites/extension-dev-guide/content/zh/extension-customization/access-control.md [hunk 1]
```diff
@@ -3,5 +3,284 @@ title: 访问权限控制
...
+- scope.kubesphere.io/workspace 
+- scope.kubesphere.io/namespace
+- scope.kubesphere.io/devops
+
+以上五个标签分别对应 平台，集群，企业空间，项目空间，DevOps项目。
+
+#### Role
+role 字段定义了一个完整的角色。不同层级的权限定义使用不同的类型, 其结构或借鉴或直接引用了 Kubernetes Role。如果您不熟悉 Kubernetes Role，请参考 
+[Role and ClusterRole](https://kubernetes.io/zh-cn/docs/reference/access-authn-authz/rbac/#role-and-clusterole) 。 
+有以下几种不同类型可选：
```
Comment (benjaminhuo) [inline sites/extension-dev-guide/content/zh/extension-customization/access-control.md:86]: should be changed to below?
- GlobalRole
- ClusterRole
- WorkspaceRole
- ProjectRole

Comment (zhou1203) [reply sites/extension-dev-guide/content/zh/extension-customization/access-control.md:88]: Fixed

File: sites/extension-dev-guide/content/zh/extension-customization/access-control.md [hunk 2]
```diff
@@ -7,27 +7,29 @@ description: 为您的扩展组件集成 KubeSphere 的访问权限控制
...
+分层，版本控制，权限控制等管理。如果你对此还不熟悉，请参考 [KubeSphere API资源](https://todo) 。
 
-我们约定了每个资源都应该属于某一个层级，资源分层有利于我们在 KubeSphere 多租户系统中为不同租户分配资源权限，为不同资源限定了其可分配的权限边界
+我们约定了每个资源都应该属于某一个层级，资源分层有利于我们在 KubeSphere 多租户系统中为不同租户分配资源权限，为不同资源限定了其可分配的权限边界。
 我们将资源分为以下层级：
 - Global 平台
 - Cluster 集群
 - Workspace 企业空间
 - Project 项目空间
-- DevOps 项目
```
Comment (benjaminhuo) [inline sites/extension-dev-guide/content/zh/extension-customization/access-control.md]: - Project 项目
- DevOps DevOps 项目

Comment (benjaminhuo) [reply sites/extension-dev-guide/content/zh/extension-customization/access-control.md]: @zhou1203

Comment (zhou1203) [reply sites/extension-dev-guide/content/zh/extension-customization/access-control.md]: Thanks for your suggestion! It`s very useful.

**Pull request merged**